scripts/apps/search/directives/ItemList.js

```diff
@@ -493,6 +493,9 @@ export function ItemList(
 
             var ListPriority = function(props) {
                 var css = {className: 'list-field urgency'};
+                if (config.reducedItemListRowHeight) {
```

Comment:
I really liked the idea you have suggested here -  I have renamed the config to config.list.thinRows ( I just thought 'narrow' would indicate narrowing the vertical dimension of the list more than reducing the row height ). Please feel free to comment if you think 'narrow' would still be a better option to go with - will be absolutely happy to change it.